docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md

```diff
@@ -142,44 +226,13 @@ use Spryker\Zed\TaxApp\Communication\Plugin\Oms\Command\SendPaymentTaxInvoicePlu
 
 ```
 
-### 3. Configure Calculation Dependency Provider
-
-Add the following to `src/Pyz/Zed/Calculation/CalculationDependencyProvider.php`:
-
-```php
-// ...
-
-use Spryker\Zed\TaxApp\Communication\Plugin\Calculation\TaxAppCalculationPlugin;
-
-// ...
-
-    /**
-     * @param \Spryker\Zed\Kernel\Container $container
-     *
-     * @return array<\Spryker\Zed\CalculationExtension\Dependency\Plugin\CalculationPluginInterface>
-     */
-    protected function getQuoteCalculatorPluginStack(Container $container): array
-    {
-        /** @var array<\Spryker\Zed\Calculation\Dependency\Plugin\CalculationPluginInterface> $pluginStack */
-        $pluginStack = [
-            // ...
-        
-            // Please put this plugin after all other tax calculation plugins.
-        
-            new TaxAppCalculationPlugin(),
-        
-            // ...
-        ];
-        
-        return $pluginStack;
-    }
+## Integrate the Vertex app
 
-// ...
-```
+Spryker does not have the same data model as Vertex requires to do correct taxes calculation, so the integration requires from the project developers to add some missing information to the Quote object before sending calculation request.
 
-## Integrate the Vertex app
+ [image here]
```

Comment:
![image](https://github.com/spryker/spryker-docs/assets/454530/b8afd6e4-2579-40a2-823e-4269a8370604)

Reply:
@lenadoc please add it this picture instead of `[image here]`